modules/rpc/thrift_test.go

```diff
@@ -27,18 +27,43 @@ import (
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 
+	"go.uber.org/fx/config"
 	"go.uber.org/fx/modules"
 	"go.uber.org/fx/service"
 	"go.uber.org/yarpc/api/transport"
 	"go.uber.org/yarpc/encoding/thrift"
 )
 
+type testHost struct {
```

Comment:
instead add a NopHostWithConfig or something similar in service/host_mock.go. This will be useful for other tests too.